manifest.toml

```diff
@@ -78,6 +78,13 @@ ram.runtime = "50M"
     choices = ["always", "errors_only", "never"]
     default = "errors_only"
 
+    [install.healthcheck_url]
+    ask.en = "Optional - you may specify a healthcheck URL to check against silent failures."
+    ask.fr = "Optionnel - vous pouvez spécifier une URL de vérification de l'état pour vérifier les pannes silencieuses."
+    type = "string"
+    default = ""
+    optional = true
+
```

Comment:
If that's okay I would remove this from the install questions and only keep this in the config panel because that sounds like a more advanced-ish feature (or at least likely to confuse newcomers? considering the app ain't 100% stragithforward in its current state)

Reply:
```suggestion
```